src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -25,16 +25,18 @@
                 <shared:commentsPanel commentsForStudentsTables="${data.commentsForStudentTable}" courseId="${data.courseId}" forRecordsPage="${true}" />
                 <br>
                 <c:forEach items="${data.sessionNames}" var="fsName" varStatus="fbIndex">
-                    <div class="well well-plain student_feedback"
+                    <div class="student_feedback panel panel-default"
```

Comment:
Class names should be delimited by hyphens rather than underscores. Do address this issue in the other parts of the changes that you made.